mujoco_warp/_src/passive.py

```diff
@@ -185,6 +271,34 @@ def passive(m: Model, d: Data):
     inputs=[m.dof_damping, d.qvel, d.qfrc_spring],
     outputs=[d.qfrc_damper, d.qfrc_passive],
   )
+  wp.launch(
```

Comment:
Kernel launches have a small overhead cost.  let's do something like:

```py
if m.nflex:
  wp.launch(_flex_elasticity, ...
```

Also it looks like `_damper_passive` already writes to `qfrc_passive`.  It's good to reduce kernel launches and avoid writing back out to global memory where possible.  Do you think you could move the `_flex_elasticity` launch after `_spring_passive` but before `_damper_passive`?

That way, you can delete `_combine_qfrc` and instead `_damper_passive` will write the sum to `qfrc_passive`

Reply:
Actually ignore the first part of this suggestion.  I just remembered that if any of the launch dims are zero, Warp is smart enough to skip enqueuing a kernel launch, so because `n.flexelem` == 0 we are OK, you don't need the if.

I would still do the second part of my recommendation, which is remove the `_combine_qfrc`

Reply:
I didn't realize the order mattered, thanks! Done.